backend/Routes/Controllers/group/index.js

```diff
@@ -0,0 +1,167 @@
+const express = require("express");
+const router = express.Router(); 
+const groups = require(".../../../Models/Groups/group-model");
+
+const validator = require("../../../Schema");
+const groupSchema = require("../Schema/groups/groups");
+
+router.post("/addGroup", (req, res) => {
+  let validate = validator.compile(groupSchema.addGroup);
+  let valid = validate(req.body);
+  if (!valid) {
+    res.status(200).json({
+      success: false,
+      error: validate.errors.reduce
```

Comment:
You can create a helper function for this thing and export it to other file. so that all developers can use the function.